src/routes/rules/RulesView.vue

```diff
@@ -1,289 +1,374 @@
 
-      <!-- One-Offs -->
-      <v-row>
-        <div class="d-flex">
-          <v-icon
-            size="x-large"
-            color="black"
-            class="mr-4"
-            icon="mdi-delete"
-            aria-label="one-off icon"
-            aria-hidden="false"
-            role="img"
-          />
-          <h1 class="gradient-text">
-            {{ t('rules.oneoffs.title') }}
-          </h1>
-        </div>
-        <p>
-          {{ t('rules.oneoffs.text') }}
-        </p>
-      </v-row>
-      <v-row
-        v-for="(ruleRow, rowIndex) in oneOffs"
-        :key="`one-off-row-${rowIndex}`"
-        align="start"
-        class="my-6"
-      >
-        <v-col
-          v-for="(rule) in ruleRow"
-          :key="rule.title"
-          md="6"
-          sm="12"
-          class="my-4"
-        >
-          <RulePreview
-            ref="preview"
-            :title="t(rule.title)"
-            :description="t(rule.description)"
-            :animated-img="rule.animatedImg"
-            :static-img="rule.staticImg"
-            :icon="rule.icon"
-            @animate="handleAnimate"
-          />
-        </v-col>
-      </v-row>
+          <v-row>
+            <h1 class="text-h2 text-surface-2 mt-5 section-title">
+              {{ t('rules.actions') }}
+            </h1>
+          </v-row>
+          <v-row
+            v-for="(ruleRow, rowIndex) in rules"
+            :key="`rule-row-${rowIndex}`"
+            align="start"
+            class="my-6"
+          >
+            <div v-for="rule in ruleRow" :key="rule.title" class="flex-column">
+              <div class="d-flex">
+                <v-img
+                  :src="rule.staticImg"
+                  class="mr-1 max-w-24"
+                  aria-hidden="false"
+                  role="img"
+                />
+                <h1>
+                  {{ t(rule.title) }}
+                </h1>
+              </div>
+              <p class="d-block">
+                {{ t(rule.description) }}
+              </p>
+              <p v-if="rule.title === 'rules.scuttle'" class="d-flex my-5">
+                <v-img
+                  src="../../../public/img/rulesView/rules_action_clubs.svg"
+                  class="mr-1 max-w-24"
+                  aria-hidden="false"
+                  role="img"
+                />
+                &nbsp; Clubs (weakest) &lt; &nbsp;<v-img
+                  src="../../../public/img/rulesView/rules_action_diamond.svg"
+                  class="mr-1 max-w-24"
+                  aria-hidden="false"
+                  role="img"
+                />
+                &nbsp; Diamonds &lt; &nbsp;<v-img
+                  src="../../../public/img/rulesView/rules_action_hearts.svg"
+                  class="mr-1 max-w-24"
+                  aria-hidden="false"
+                  role="img"
+                />
+                &nbsp; Hearts &lt; &nbsp;<v-img
+                  src="../../../public/img/rulesView/rules_action_spades.svg"
```

Comment:
Images don't need the public path. We have them set up to just start with `/img` etc. This is managed via vite config for local development and sails config for production
```suggestion
                  src="/img/rulesView/rules_action_clubs.svg"
                  class="mr-1 max-w-24"
                  aria-hidden="false"
                  role="img"
                />
                &nbsp; Clubs (weakest) &lt; &nbsp;<v-img
                  src="/img/rulesView/rules_action_diamond.svg"
                  class="mr-1 max-w-24"
                  aria-hidden="false"
                  role="img"
                />
                &nbsp; Diamonds &lt; &nbsp;<v-img
                  src="/img/rulesView/rules_action_hearts.svg"
                  class="mr-1 max-w-24"
                  aria-hidden="false"
                  role="img"
                />
                &nbsp; Hearts &lt; &nbsp;<v-img
                  src="/img/rulesView/rules_action_spades.svg"
```